### ISSUE TITLE
Multiple Files

### ISSUE THREAD

**Issue by SamParisot** [nspire-emus/firebird#126]
How can I transfer an entire directory of files at the same time without doing it 1 by 1?

**Comment by Vogtinator** [comment]
Using Android that's not possible yet.
I implemented a hacky workaround for you to test: A new button which transfers a directory recursively.
Simply select a file within the directory you want to transfer the contents of and it should work.

[firebird-emu.zip](https://github.com/nspire-emus/firebird/files/2012075/firebird-emu.zip)

**Comment by SamParisot** [comment]
I'll test it immediately

**Comment by SamParisot** [comment]
For a directory with a smaller amount of files (less than 40 I'd say), this button works great. However, I'm transferring over 300 files. It may be because I am transfering sprites that are usually less than a megabyte and it overloads the calculator and/or emulator which causes a crash. After said crash if you hit resume to load your save point before the transfer it goes straight to debugging mode, so you have to reboot the Ti-NSpire.

I would debug it and send a screenshot if I had my linux PC with me.

I'm trying to get a port of Cave Story since it's basically what I use as a "universal test game" so that's why it's over 300 files - sprites

**Comment by Vogtinator** [comment]
Well, the Nspire simply can't cope with that many files. Not something firebird can fix.
I suggest to pack it into a tar or other kind of archive and implement file access yourself instead.

**Comment by SamParisot** [comment]
Yeah I didn't think so. I would do that but I honestly am not sure how. I think I'll just split up the files into folders of 30 or 40 on my phone and copy them into the sprite directory on the calculator.
